spec/raven/event_spec.rb

```diff
@@ -347,6 +347,7 @@ def raven_context
   end
 
   describe '.initialize' do
+    # NOTE: This test is completely pointless.
```

Comment:
There's https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher for that.